### PR TITLE
docs: Use camel case for json fields

### PR DESCRIPTION
**Pull request by jerargus**
Closes: #world-414

## What is the purpose of the change

Update documentation to use camelCase for JSON fields. The relevant code has already been updated.

Update a field in Nakama storage to use camelCase instead of snake_case to keep things consistent.

NOTE: The Nakama change is NOT backwards compatible; existing projects will need to wipe their Nakama postgres storage to use this new image. My understanding is that DF is the main project that this will impact, and they tend to wipe their storage about once a week anyway.

## Testing and Verifying

This change is a trivial rework/code cleanup without any test coverage.

### CODE REVIEW

File: relay/nakama/main.go [hunk 1]
```diff
@@ -44,7 +44,7 @@ const (
 	EnvCardinalAddr      = "CARDINAL_ADDR"
 	EnvCardinalNamespace = "CARDINAL_NAMESPACE"
 
-	cardinalCollection = "cardinal_collection"
+	cardinalCollection = "cardinalCollection"
```
Comment (jerargus) [inline relay/nakama/main.go:47]: NOTE: While most of this PR is just documentation changes, this is a breaking change for any Nakama users.

**Comment by technicallyty** [comment]
changing title to docs since its mainly that

**Comment by technicallyty** [comment]
@heronimus docs deploy failing here

**Review by technicallyty — APPROVED** [review]
LGTM

**Review by namrapatel — APPROVED** [review]

**Pull request merged**